main/webapp/modules/core/scripts/index/open-project-ui.js

```diff
@@ -124,21 +163,62 @@ Refine.OpenProjectUI.prototype._buildTagsListPanel = function() {
 };
 
 
-Refine.OpenProjectUI.prototype._fetchProjects = function() {
-    var self = this;
-    $.ajax({
-            type : 'GET',
-            url : "command/core/get-all-project-metadata",
-            dataType : 'json',
-            success : function(data) {
-                    self._renderProjects(data);
-                    self.resize();
-            },
-            data : {},
-            async : false
+Refine.OpenProjectUI.prototype._searchAnimation = function() {
+    var search = $('#searchIcon');
+    var form = $('.header-search-box');
+    var icon = $('.magnifying_glass');
+    search.click(function () {
+        if (form.is(':hidden'))
+        {
+            $("#tagsUl").hide()
+            $("#divInput").show()
+            icon.addClass("magnifying-glass-open")
+            form.show()
+            form.focus()
+        }
+        var widthFormOpen = Math.floor($('#right-panel-body').width() * 4 / 5);
+        form.animate({
+            'width': form.width() == widthFormOpen ? '0px' : widthFormOpen + "px"
+        }, 'fast', function () {
+            if (form.width() == 0) {
+                form.hide()
+                form.val('')
+                icon.removeClass("magnifying-glass-open")
+                $("#tableBody").filterListSearch("")
+                $("#divInput").hide()
+                $("#tagsUl").show()
+            }
+        });
     });
 };
 
+Refine.OpenProjectUI.prototype._searchInput = function() {
+    var search = $('#searchInProjects');
+    //setup before functions
+    var typingTimer;                //timer identifier
+    var doneTypingInterval = 2000;  //time in ms, 2 seconds
```

Comment:
```suggestion
    var doneTypingInterval = 1000;  //time in ms, 1 seconds
```
After testing this interactively, 2 seconds feels a bit long to me, I would expect something a bit faster.